CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.16)
 project(backscrub CXX)
 
+if(NOT DEFINED TENSORFLOW)
+  set(TENSORFLOW tensorflow)
```

Comment:
Consistent naming with `Makefile`? --> `TFBASE`?

Reply:
Can do - might go the other way?

Reply:
Just important to try to have both in sync. Direction doesn't matter.